modules/cluster-logging-clo-status.adoc

```diff
@@ -118,6 +122,8 @@ The following are examples of some condition messages from the `Status.Nodes` se
 
 A status message similar to the following indicates a node has exceeded the configured low watermark and no shard will be allocated to this node:
 
+.Example output
```

Comment:
This doesn't seem to relate to any command output, though. Maybe this header is fine anyway.

Reply:
These are various outputs the user might see in the `status` field when running: `oc get clusterlogging instance -o yaml`